make.js

```diff
@@ -230,15 +230,13 @@ target.build = function() {
                         console.log('> getting module externals');
                         getExternals(modMake.externals, modOutDir);
                     }
+                }
 
-                    if (mod.type === 'node' && mod.compile == true) {
-                        var commonPack = util.getCommonPackInfo(modOutDir);
-
-                        // assert the pack file does not already exist (name should be unique)
-                        if (test('-f', commonPack.packFilePath)) {
-                            fail(`Pack file already exists: ${commonPack.packFilePath}`);
-                        }
+                if (mod.type === 'node' && mod.compile == true) {
```

Comment:
Will making this consistent with the above condition work?

```
if ((mod.type === 'node' && mod.compile == true) || test('-f', path.join(modPath, 'tsconfig.json'))) {
```

I would rather this be consistent.

Reply:
Yep that still works, I've added it in